matlab-src/whole_body_model_functions/wbm_getWorldFrameFromFixedLink.m

```diff
@@ -0,0 +1,33 @@
+function [ world_R_base,world_p_base ] = wbm_getWorldFrameFromFixedLink( varargin )
```

Comment:
The balancing code and almost all wbm functions use linear-angular. I noticed that if "normal mode" for wbm function is used, wbm_setWorldFrame is actually useless and can be removed from balancing code. I think this can avoid potential convention errors. 
I'm going to test the balancing code today,  if @naveenoid finished the modifications. 
I also modified the feet orientation to avoid angular velocity integration.